site/components/Text/Text.tsx

```diff
@@ -18,6 +18,7 @@ type Props = {
     | 'span';
   children?: React.ReactNode;
   color?: BoxProps['color'];
+  display?: BoxProps['display'];
```

Comment:
Adding `display` to `Box` shouldn't be needed anymore, right?

Reply:
It seems to me that since the heading elements are block level, it needs to keep `display` or add `display="inline-flex"` to the `Link`?

Reply:
Oh sorry, i misread, i thought this was `Box`, but it's `Text` 😅